packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/telephony-line-phone-accessories.routing.js

```diff
@@ -0,0 +1,19 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'telecom.telephony.billingAccount.line.phone.accessories',
+    {
+      url: '/accessories',
+      views: {
+        'lineView@telecom.telephony.billingAccount.line':
+          'telephonyLinePhoneAccessories',
+      },
+      resolve: {
+        billingAccount: /* @ngInject */ ($transition$) =>
```

Comment:
Isn't this already injected in some parent state ? In that case it will be inherited, no need to declare it again

Reply:
I don't think so.

Reply:
I think @marie-j talking about this one : https://github.com/ovh/manager/blob/master/packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/billingAccount.routing.js#L22

Reply:
OK, I test what you asked and it's not already injected. You can access to this accessory view from the phone list actions view. Which not inject the billingAccount in the state.